### PR TITLE
feat: add additional release links to the release

### LINKED ISSUE #281: Append the release channels to the release
Is there a possibility to append the release channels to the release?
Atm the only possibility to see the release channels is on a PR/Issue.

I have 4 release channels internally that I use (`S3`, `Docker`, `Pypi`, `GithubRelease`)
It would be nice to see the other releases too.

### PR DESCRIPTION
**Pull request by Chumper**
A new option `addReleases` has been added.
Setting this option will instruct the plugin to append all
additional releases to the Github release on the top.

Closes #281

Screen Shot 2020-09-01 at 3 17 29 AM

### CODE REVIEW

**Comment by Chumper** [comment]
I am not sure about the travis error. This seems to be a generic problem, maybe with NPM? 
Locally I have 100% coverage.
Screen Shot 2020-09-01 at 3 21 30 AM

**Comment by Chumper** [comment]
@gr2m @pvdlg Could you help with the travis errors here? I am not sure on how ti fix them.

**Comment by felipecrs** [comment]
I believe you can add the closure of https://github.com/semantic-release/semantic-release/issues/1617 to this PR as well. (Closes https://github.com/semantic-release/semantic-release/issues/1617)

**Comment by gr2m** [comment]
I've restarted the CI builds, but the same tests failed again. I'll need some time to investigate, and I don't know when I'll have that time right now, I'm afraid. We didn't have builds for a while, so it might be a change in some of the dependencies ...

**Comment by Chumper** [comment]
I will also have a look tomorrow, I just have never seen that the travis files are in a different repository.

**Comment by Chumper** [comment]
I found the error. I was using a feature that is not supported in node 10 and 12.
I was using `Optional chaining` like this `property?.id` which is not supported in node version below 13.

**Review by gr2m — APPROVED** [review]
<a href='https://hearts.dev/projects/3/users/gr2m'></a>

Great work, thank you!

**Comment by gr2m** [comment]
Actually I wonder if we should add the links to the bottom, not the top. I think it will work better, because changelog is the primary information I expect in release notes, links to sibling releases is meta information

**Comment by Chumper** [comment]
I would argue against that, the release information is always short enough so that you can skip those lines and get to the changelog.
The changelog could contain a few more lines and at least in my case I am more interested in the releases than the changelog.

If you give me another hour, I can change this so that you could do: `addReleases: <top|bottom|false>`

**Comment by felipecrs** [comment]
I agree that the release links should be at the bottom and not in top.

**Comment by Chumper** [comment]
Sure, I will rework it to be false by default, but also to accept either `bottom` or `top` and adjust the code accordingly.

**Comment by Chumper** [comment]
@gr2m @felipecrs I have added the discussed changes. Allowed values are now `false`, `top` or `bottom`.

Default will be `false`.

Please review again and merge if you think this will work.

File: README.md [hunk 1]
```diff
@@ -204,3 +204,13 @@ Each label name is generated with [Lodash template](https://lodash.com/docs#temp
 The `releasedLabels` ```['released<%= nextRelease.channel ? ` on @\${nextRelease.channel}` : "" %> from <%= branch.name %>']``` will generate the label:
 
 > released on @next from branch next
+
+#### addReleases
+
+When set this is option will append all additional releases except the github release to the body of the github release.
```
Comment (gr2m) [inline README.md:210]: I'd rephrase the sentence, now that it can be added to the top or bottom

```suggestion
Add links to other releases to the GitHub release body.
```

File: README.md [hunk 2]
```diff
@@ -204,3 +204,13 @@ Each label name is generated with [Lodash template](https://lodash.com/docs#temp
...
+
+Valid values for this option are `false`, `top` or `bottom` where the releases will be added on the top when set to `top` and on the bottom when set to `bottom`.
```
Comment (gr2m) [inline README.md:212]: I think `top` and `bottom` are self-explanatory enough? I would add quotes to make clear that `false` should not be a string.

```suggestion
Valid values for this option are `false`, `"top"` or `"bottom"`.
```

**Comment by Chumper** [comment]
Short and informative, I applied the changes.

**Review by gr2m — APPROVED** [review]
Thank you Nils, this is great

**Comment by felipecrs** [comment]
This is awesome!

**Review by blg-666 — APPROVED** [review]
